tests/test_dataobject.py

```diff
@@ -0,0 +1,79 @@
+import numpy as np
+import pyvista
+from pyvista import examples
+
+
+def test_eq_wrong_type(sphere):
+    assert sphere != [1, 2, 3]
+
+
+def test_uniform_eq():
+    orig = examples.load_uniform()
+    copy = orig.copy(deep=True)
+    copy.origin = [1, 1, 1]
+    assert orig != copy
+
+    copy.origin = [0, 0, 0]
+    assert orig == copy
+
+    copy.point_arrays.clear()
+    assert orig != copy
+
+
+def test_polydata_eq(sphere):
+    sphere.clear_arrays()
+    sphere.point_arrays['data0'] = np.zeros(sphere.n_points)
+    sphere.point_arrays['data1'] = np.arange(sphere.n_points)
+
+    # error here:
+    # was = sphere.point_arrays.active_scalars_name
+    # sphere.active_scalars_info
+    # now = sphere.point_arrays.active_scalars_name
+    # assert was == now
```

Comment:
Is this comment meant to be kept? What does it convey?

Reply:
This comment was here merely for debugging.  Thanks for catching it, I'll remove.